documentation/adr/0002-wiki-platform.md

```diff
@@ -0,0 +1,294 @@
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Usability:** Non-technical users should be able to access and create content with minimal training or guidance.
+- **Content Review:** Collaborators should be able to review and edit draft content before those changes are published.
+- **Version History:** Editors should be able to see and restore previous versions of a given page.
+- **Multi-Media:** The platform should support multiple types of media (e.g. videos, images, file uploads, tables, diagrams) with minimal configuration.
+- **Internationalization (i18n):** The platform should provide support for displaying content in multiple languages.
+- **Web Analytics:** The platform should provide support for tracking site usage and other web analytics.
+- **Onboarding Costs:** Onboarding new members to the platform should be relatively inexpensive, both in terms of staff time/resources and direct costs (e.g. licensing fees).
+- **Maintenance Costs:** It should not be prohibitively expensive to maintain the wiki, both in terms of staff time/resources and direct costs (e.g. hosting fees).
+
+#### Nice to Have
+
+- **External Contributions:** Members of the public should be able to suggest changes to wiki content and internal stakeholders should be able to review those contributions before they are published.
+- **Data Access:** Content generated and stored in the wiki should be accessible outside of the wiki platform, either through syncing content to an HHS owned repository or through an official API.
+- **Open Source:** The tool used to manage and host the wiki content should be open source, if possible.
+
+## Options Considered
+
+- [Confluence](confluence)
+- [Notion](notion)
+- [GitHub Wiki](gh-wiki)
+- [GitBook](gitbook)
+- [WikiJS](wiki-js)
+- Git-based Headless CMS (e.g. [Decap CMS](git-cms-decap) and [Tina.io](git-cms-tina))
+- API-based Headless CMS (e.g. [Strapi](strapi) and [Directus](directus))
+
+## Decision Outcome <!-- REQUIRED -->
+
+Chosen option: "{option 1}", because {justification. e.g., only option which meets k.o. criterion decision driver | which satisfies x condition | ... }.
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- {e.g., improved performance on quality metric, new capability enabled, ...}
+- ...
+
+### Negative Consequences <!-- OPTIONAL -->
+
+- {e.g., decreased performance on quality metric, risk, follow-up decisions required, ...}
+- ...
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+### Confluence
+
+[Confluence](confluence) is a Software as a Service (SaaS) documentation and collaboration tool offered by Atlassian that organizes content into "spaces" and offers a series of templates and components that can be used to create custom documentation for internal and external stakeholders.
+
+#### Details
+
+- **Hosting:** SaaS
+- **Pricing:** [$5.75 (standard) or $11 (premium) per user per month](confluence-pricing)
```

Comment:
Is this the cost even just to view content, or for edit users only?

Reply:
All of the pricing is for named users, what I was imagining would be HHS staff and contractors whom we want to have direct edit and content moderation access. 

The ability for members of the public to view, comment on, and suggest edits is covered by the external contributions section and in the Pros and Cons list, which often references features like publishing spaces or pages to the web. But I can definitely make that distinction a bit clearer!

Reply:
Making that distinction clearer, in this PR or the next one, would be great! Thx!

Reply:
File moved and comment addressed in #30